### ISSUE TITLE
Ask HN: Cheap places to live with a good intellectual atmosphere?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/home#17]
![image](https://user-images.githubusercontent.com/194400/46596910-e2fee300-cad5-11e8-9b14-c4ec234a0b1f.png)
https://news.ycombinator.com/item?id=18164189
> "_I'm a software engineer in the bay area and have been thinking about moving somewhere cheaper (in USA or aboard) where I can live cheaply and focus on my own intellectual pursuits. I'd love to be in a place where the living costs are low and where there exists a thriving intellectual community (I've noticed cities near top academic institutions tend to create that sort of atmosphere but not necessarily). I'd prefer a place where English speakers are common enough such that I won't feel isolated by a language barrier. I haven't traveled too often and would love to hear from HN community about any places that match this general description. Thank you in advance!_"

**Comment by nelsonic** [comment]
_Top_ comment:
![image](https://user-images.githubusercontent.com/194400/46598039-789c7180-cada-11e8-8d5f-1bc1a186d24a.png)

@iteles people _want_ this!!

**Comment by mhoad** [comment]
For the record I just discovered home from that same thread. I've had some experience with some similar concepts such as sende.co as well as spending a decent amount of time in Portugal over the past few years. I spent this morning reading over the plans you've outlined in some detail and I am really excited to see where you end up taking this project. 

I am officially making the move to Portugal next year and picking up residency in the process. I'd love to come and take a look in person and see if I can't make myself useful for the project in some capacity as well.

**Comment by nelsonic** [comment]
@mhoad, stoked that you found the project via the thread, 
I was _definitely_ not expecting it to "blow up" like that...! 😮 

We are still in "stealth mode" until we are able to _open_ the house, 
but as soon as it's open we will send you a way to "subscribe" for updates 
and _contribute_ with your insights/experience [_which is very much welcome!_]

The plan in this repo is just a "stream of consciousness" ...
Sorry I _wasted_ your time with my "ramblings" ...
We do have a _decent_ plan for what to do with the house.
And we will _very much_ be welcoming input from the community to make it better!

We are hoping to create an **Open Source _Community_** that everyone can contribute to.
And if people decide to "fork" it and do something different, 
e.g: because they "disagree" with something we aren't prepared to "budge" on,
then _anyone_ is welcome to use _all_ of the "intellectual property" and _run_ with it!

When you come to Portugal next year, we look forward to meeting you IRL! 🎉

**Comment by stefek99** [comment]
**Well done.**

Also arrived from HN.

Also rabbit hole reading all the plans.

**Swimming pool and functional fitness is badass amazing.**

A little bit burnt with @astralship - _"realising human potential to heal the planet"_

Now back in London, selling time for money as a contractor, restoring savings and financial stability.

Winter hopefully in Asia, Spring hopefully in Braga?

Been to Portugal a number of times, including 4x Boom Festival.

Interested in areas such as biohacking, nootropics, microdosing, psychedelics - to have the optimised baseline and then go even further...

**Comment by nelsonic** [comment]
@stefek99 it's only "well done" when we _open_. (_right now it's just a "building site"!_) ⚠️ 🚧
It feels like this has been "in progress" for _ever_! ⏳ 
To give you an idea @iteles first submitted our "planning permission" 10 months ago and each time they take 6-8 weeks to ask us to complete "_one more form_" ... it's _exhausting_! Hardware is _hard_!! 🙄 
We had to submit 12 different plans for everything from AirCon to Waste Water Treatment, Network Topology and Predicted Energy Consumption ... 📝 
it's a good "forcing function" to ensure that we consider all aspects 🤔 
but it sometimes feels like the authorities are making our lives _needlessly_ difficult & expensive! 💸 

We're _excited_ to start sharing more about the idea(s) v. soon.
We are just _weary_ of sharing "too much" detail and risk sounding like "dreamers" or "vapourware". 

I _really_ wasn't expecting anyone **`else`** to read this content, much less _interact_ with it. [yet!] 😮 
It's a _welcome_ surprise! 😜

**Comment by stefek99** [comment]
I cannot believe you actually applied for all the planning permissions!

I guess you are serious about business and longevity.

Me = hacker approach.

-----

Look how little:

![image](https://user-images.githubusercontent.com/249703/46684436-b25c9d80-cbea-11e8-87b7-b26ddfb4995c.png)

https://www.youtube.com/watch?v=_6YtmPo9pxg

**Comment by nelsonic** [comment]
@stefek99 we would _love_ to take the "hacker" approach  ... however would get _shut down_ and lose _all_ our time/investment _very_ quickly if we didn't have all the paperwork in order. 
There's no getting around it. 
If we were based in the mountains we could do whatever we want and nobody would care/notice. 
In he city we have _neighbours_ who while "friendly" could _easily_ "embargo" our entire project with a single noise complaint. 🙄

**Comment by Joshfairhead** [comment]
Cool plans! Appreciate the discussion, Portugal is nice. I lived in a tourist trap in the Algarve for 8months - prices were high. The north sounds really nice though :)

**Comment by 2color** [comment]
@nelsonic Can you elaborate more on the planning permissions? 

How is it compared to other places in your experience and based on the research you've done?

**Comment by nelsonic** [comment]
Hi @2color, planning permission is _always_ tedious in _any_ country unless you are _wealthy_/_powerful_. 🙄 

@iteles put in the most work to secure the planning permission, 
I just did a few sketches of ideas and attended meetings to discuss with Architects ... 
we paid experienced local architects (a _chunk_ of cash) to finalise our ideas/plans 
(_planning permission cannot be submitted to the authorities without a signature from an Architect..._)
We had to re-submit plans several times and our biggest request was "_rejected_" (see below)

For reference, we recently visited the Dock Inn Shipping Container Hotel https://www.dock-inn.de 
in Warnemünder, Germany:

![dock-inn-front](https://user-images.githubusercontent.com/194400/47272337-0c127f80-d57c-11e8-97fc-de99013d948d.jpg)

We spoke with the _seasoned_ developers who informed us that it took 3 years from concept 
to breaking ground on the project ... roughly **18 months** just to get **planning permission** on virgin land.

# tl;dr

We _often_ hear stories of people with "_influence_" in Portugal either getting their planning requests "_fast tracked_" or simply not _bothering_ with permission and paying a small fine (_relative to their wealth or the cost of the project_) ... very occasionally people have their projects "embargoed" but then pay someone somewhere and their problems magically disappear... 🙊 

> **Note**: Portugal has _relatively **really low**_ "corruption" from an international perspective 
https://en.wikipedia.org/wiki/Corruption_in_Portugal and we are _not_ "accusing" anyone in the council/government of any "wrongdoing" (_from a **legal** perspective..._) 
Simply that from _experience_ we feel that the _system_ is not designed for small business innovation or progress in affordable housing. 

> For example: we spent several weeks preparing the designs and planning application for converting the roof into a Roof Garden: https://github.com/nelsonic/roof-garden 
From this "_boring_" standard tiled roof:
![image](https://user-images.githubusercontent.com/194400/47266024-80233800-d528-11e8-915a-93cca9fcf3ec.png)
To an open space with full day sunlight and wheelchair access:
![image](https://user-images.githubusercontent.com/194400/47266032-b82a7b00-d528-11e8-9e5c-dc70401685ec.png)

> The roof garden would have more than doubled our useable out-door space and had zero impact on the neighbours visibility or sunlight. 
But after "processing" (_sitting on!_) our request for 6 months the council _rejected_ it saying we aren't "allowed" to change the roof. We aren't even allowed to change **broken** roof tiles without permission!
It's not like the building is "listed" or of any architectural/cultural significance ... 😕 
it's an _abandoned_ residential building that has been sitting empty for several years! 
The council planning committee is just fixed/closed mindset and oppose _any_ kind of change.
![image](https://user-images.githubusercontent.com/194400/47265893-d1322c80-d526-11e8-8d9f-3c9b60b4d1ee.png)
They approve new shopping malls (_like we need any more!_) because the mega companies wine-and-dine the officials. But increasing the supply of affordable housing with great features, nope!

## An estimated 735,000 properties are left empty in Portugal
https://www.dailymail.co.uk/news/article-4028126/Portugal-mansions-sit-abandoned-photos-reveal-dilapidated-remains-half-lived.html
With a population of 10M having 735k empty houses is a _significant_ percentage of the housing stock! 

> There is a _desperate_ housing shortage in Portugal that has lead to **9.5% rise** in **house prices**!
http://www.theportugalnews.com/news/portugal-on-track-for-biggest-property-price-rise-in-europe/46741
Instead of making it _easy_ for people to renovate old/abandoned houses and make them available to the market, the authorities drag their feet on reviewing planning and ultimately reject requests without giving _good_ reasoning.

Planning permission in other countries is similar. The authorities are rarely "pro change".
Their intentions might be good e.g: protect the aesthetic of the town/city/street or avoid building code violations. But ultimately they stand in the way of people wanting to make progress.

If you (_or anyone **`else`**_) are thinking of renovating an older building in Portugal or another "older" country where pointless rules and red-tape prevail over common sense,
my advice is: ***don't***. 
The (_intentional_) bureaucracy is _demoralising_. 
If you still think you can do it, double your financial/time budget/estimate 
and allow for at _least_ 8 months of back-and-forth between the planning authorities.
Remember that until the council approves construction no Bank will loan you a cent ... 
Nobody will care/cry if you run out of cash before completing the project.
There are more than enough abandoned/half completed building projects in Portugal
testament to the fact that the housing market is "_broken_"!
https://www.idealista.pt/en/news/financial-advice-portugal/2018/03/28/62-there-are-least-44000-dilapidated-buildings-lisbon

If that does not put you off _completely_ then pick a _good_ architect and pay them well 
(_contingent on project approved and completion!_) so they don't "lose interest" in your project while waiting for the planning permission or during construction.

> We are doing **`home`** because we are on a _mission_. 
We won't be stopped by some paper-pushing 9-to-5 bureaucrat who has zero personal/financial incentive to change or allow anyone _else_ to make progress!
We definitely aren't doing this for a financial return/gain, if we wanted to make money we would invest in tech stocks and double our money _much_ faster. We _sold_ stocks to pay for the construction... 😞 💸 

Anyway, sorry for the "rant". If you have a building project that you are _passionate_ about, try to take the _emotion_ out of it, take a step back and think through how much pain you will go through. If you still think it's worth it, make sure you have the capital and time commitment. 
If you want _way_ more detail, we will be publishing everything in a free PDF after we open. 📘

**Comment by nelsonic** [comment]
Closing this issue as it was simply to inform of the Ask HN Question. ✅

**Comment by iteles** [comment]
@2color If you'd like some more info on the 'nuts and bolts' of planning permissions, please open an issue, I'm happy to fill you in! 😊

**Comment by stefek99** [comment]
One should understand the mindset of someone employed as a beaurocrat, someone earning minimal wage _(or slightly above)_ having little or no prospect of other employment.

Upside of accepting something new - **none**

Donwside of accepting somethign new - **non non-existent** _(double negation)_

Meeting with local officials and explaining `30+` new residents generating `€€€` income and `5+` locally created jobs _(cooking, cleaning, maintenance)_ would score more points politically.

---

Related / unrelated. Some examples of massive incentives:

Puerto Rico: https://www.elvocero.com/regionales/arecibo-regala-terrenos-para-el-desarrollo-de-vivienda-y-comercios/article_97ba4de8-3675-11e8-b75c-6b33282719fb.html

Russia free land: https://www.rt.com/business/431053-russia-free-land-giveaway/

Italy free castles: https://www.independent.co.uk/travel/news-and-advice/italy-castles-villas-monasteries-give-away-free-100-historic-buildings-a7739001.html

**Comment by nelsonic** [comment]
@stefek99 _agreed_ it will be a _lot_ easier to "pitch" the _second_ house to authorities
and even get cut-price abandoned property once there's a bit of a track record.
Promising to create "good jobs" before we have good press success stories is empty.

**Comment by stefek99** [comment]
I hope there will be second, third, fourth, ..., n house all over the globe.

It is happening already, independently, so many projects we are not aware of...

I think it's easier to build from scratch than retrofit existing properties - true or false?

---

Me = wording nerd. 

Promise = likehood. 

Good jobs = any jobs. 

Bus company, taxi driver, localfarmer, hairdresser = everyone wins.

People from the entire world are likely to visit Portugal for the first time because of you, Braga building planning permission department should be made aware of the technological shift that is happening.

--- 

Full support. Keep being awesome. I'm really hoping this project will come to fruition and each next iteration will be even better...

**Comment by iteles** [comment]
@stefek99 
> I think it's easier to build from scratch than retrofit existing properties - true or false?

Definitely easier and in Portugal, most likely cheaper for the kind of thing we want to do.

The house we chose has a story and is also a great learning experience!

**Comment by 2color** [comment]
Thanks for all the input @iteles and @nelsonic.
